config/webpack.prod.js

```diff
@@ -31,6 +31,19 @@ module.exports = merge.smart(common, {
             loader: "awesome-typescript-loader",
             options: {
               declaration: false,
+              useBabel: true,
```

Comment:
Why babel and why only in prod? How you test/develop on ie11?

Reply:
What babel really does there? We have target es5 in tsconfig, so typescript prolly emits es5 and we have almost no work for babel ...

Reply:
It think there is no real advantage in adding Babel here, especially if you aim to handle IE11. You could benefit from Babel if you wish to support only the newest browsers and ship the most fitted code to them. In such case TS outputs ES6, Babel compiles only not supported syntax, so bundle is smaller, build faster, debugging easier etc. Adding IE to compilation target will force ES5 anywa, I suppose does it well already.

Instead of this, add some polyfills, maybe polyfill.io if you don't mind, and you'll be fine.